net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A small dynamic DNS client"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers \
+support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
+CHECKSUM_SHA256="23b527b7f71e1449746f1e39cbc3a074d58bd261a33399fb4e2d90eb63149059"
+SOURCE_DIR="yaddns-yaddns-$portVersion"
+PATCHES="yaddns-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+PROVIDES="
+	yaddns$secondaryArchSuffix = $portVersion
+	cmd:yaddns$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autom4te
+	cmd:automake
+	cmd:autoreconf
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:makeinfo
+	"
+
+USER_SETTINGS_FILES="
+	settings/yaddns directory
+	"
+
+defineDebugInfoPackage yaddns$secondaryArchSuffix \
+	$commandBinDir/yaddns
+
+BUILD()
+{
+	./autogen.sh
+	LIBS="-Wl,--as-needed -lbnetapi -lnetwork" runConfigure \
+		--omit-dirs binDir ./configure --bindir $commandBinDir
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	install -d $dataDir/yaddns
+	install etc/yaddns.conf $dataDir/yaddns/yaddns.conf.example
+	install -d $settingsDir/yaddns
```

Comment:
You shouldn't need to install this folder

Reply:
@alaviss `$settingsDir/yaddns` is where the user is going to put the settings, and I think we need to at least prepare the directory for them.

Reply:
It's not permitted by the [Packaging Policy][0]

> `user-settings-files`: A list of user settings file infos. The list elements must have the following format:
>
>       user_settings_file_info	::= path [ "directory" | "template" template_path ]
>
> `path` is the relative path of the settings file or directory, starting with "settings/". **It is not included in the package**. However, if `template_path` is specified, it is a path to a file included in the package that can serve as a template for the settings file. It doesn't imply any automatic action on package activation, though. If the "directory" keyword is given, the path refers to a settings directory (typical when a program creates multiple settings files). 

[0]: https://dev.haiku-os.org/wiki/PackageManagement/BuildingPackages